features/support/manual_helpers.rb

```diff
@@ -227,4 +215,70 @@ def check_for_manual_body_preview
   def check_for_clashing_section_slugs
     expect(page).to have_content("Warning: There are duplicate section slugs in this manual")
   end
+
+  def withdraw_manual(manual_title)
+    manual_id = get_id_for_manual(manual_title)
+
+    manual_services = ManualServiceRegistry.new
+    manual_services.withdraw(manual_id).call
+  end
+
+  def get_id_for_manual(manual_title)
+    visit manuals_path
+    link = page.find_link(manual_title)
+    # TODO this is pretty gross, consider making it less so
+    link.native.attribute("href").value.match(%r{\A/manuals/(.*?)(\?|\z)})[1]
```

Comment:
I agree, this isn't nice, but @evilstreak and I think that it's best tackled as part of a separate story to make all test setup use the services directly, rather than submitting forms to do it. The services would then return objects, which would make it much simpler to make assertions about changes to those objects. Let's write up a separate story to do that, and leave this as it is for now.